### DISCUSSION TITLE
how to use mimetype with a wildcard

### DISCUSSION THREAD

**Discussion by sacrosanctic** [open-circle/valibot#457] [General]
```ts
const ImageSchema = blob([
  mimeType(['image/*']),
]);
```

**Comment by fabian-hiller** [comment]
It's not currently supported. Are you sure you really want to accept every image format?

**Reply by sacrosanctic** [reply]
I was trying to mimic `<input type="file" accept="image/*" />`. Do you have an alternative, or a non exhaustive list of images to validate against?

**Reply by fabian-hiller** [reply]
I would not use a wildcard. There are many image formats that are not supported by browsers and specific operating systems. I recommend explicitly specifying only the format your system supports.

**Reply by sacrosanctic** [reply]
that is reasonable, i will do that instead
